src/components/ui/FluroTimeline.vue

```diff
@@ -25,11 +25,11 @@
 			<template v-else>
 				<fluro-infinite-scroll :perPage="1" :startPage="1" :buffer="2" :items="years">
 					<template slot-scope="props">
-						<div class="year" v-for="year in props.page">
+						<div class="year" v-for="(year, i) in props.page" :key="i">
```

Comment:
@peter-subsplash It looks like you and I did the same thing for the `v-bind:key` error, although I spelled out `index` instead of using just `i`. Curious what your thoughts are on standardizing one or the other. I think I like just `i`. Or maybe it's not necessary to choose one I'm just a big fan of consistency 😆

Reply:
Ya, either way. I usually like making things clear-and-spelled-out... but `i` is so common it's almost clearer to have it just `i`.